src/soundio/sounddevice.h

```diff
@@ -84,9 +91,21 @@ class SoundDevice {
     double m_dSampleRate;
     // The name of the audio API used by this device.
     QString m_hostAPI;
+    // The **configured** number of frames per buffer. We'll tell PortAudio we
+    // want this many frames in a buffer, but PortAudio may still give us have a
+    // differently sized buffers. As such this value should only be used for
+    // configuring the audio devices. The actual runtime buffer size should be
+    // used for any computations working with audio.
     SINT m_framesPerBuffer;
     QList<AudioOutputBuffer> m_audioOutputs;
     QList<AudioInputBuffer> m_audioInputs;
+
+  private:
+    /**
+     * The last buffer size that was sent to the `[Master],audio_buffer_size`
+     * control object. This is set in `maybeUpdateBufferSize()`.
+     */
```

Comment:
We use C style comments for disable code. Please prefer // for comments. /// is picked up by doxygen if you like.

Reply:
Yes, see:
https://github.com/mixxxdj/mixxx/wiki/Coding-Guidelines#comments